csfieldguide/general/urls.py

```diff
@@ -46,4 +47,25 @@
         "index.html",
         RedirectView.as_view(permanent=True, pattern_name="general:index"),
     ),
+    path(
+        "pixelmania-1",
+        RedirectView.as_view(
+            permanent=True,
+            url=base_url+"interactives/pixel-viewer/?colour-code=hex&hide-colour-code-picker&pixelmania&image=lake.png"
```

Comment:
Be sure to check #1304 for the true URL code that's needed for this and pixelmania-3. The example I gave there sets it to what Jack originally asked for